validator/client/src/main/java/tech/pegasys/teku/validator/client/AbstractDutyScheduler.java

```diff
@@ -54,8 +59,41 @@ public void onSlot(final UInt64 slot) {
     calculateDuties(currentEpoch);
   }
 
+  @Override
+  public void onHeadUpdate(
+      final UInt64 slot,
+      final Bytes32 previousDutyDependentRoot,
+      final Bytes32 currentDutyDependentRoot,
+      final Bytes32 headBlockRoot) {
+    if (!useDependentRoots) {
+      return;
+    }
+    final UInt64 headEpoch = compute_epoch_at_slot(slot);
+    dutiesByEpoch
+        .tailMap(headEpoch, true)
+        .forEach(
+            (dutyEpoch, duties) ->
+                duties.onHeadUpdate(
+                    getExpectedTargetRoot(
```

Comment:
Should this be:
```suggestion
                    getExpectedDependentRoot(
```
?

Seems like "dependent" and "target" roots are equivalent, if so would clarify things to use consistent naming.

Reply:
Yep, the original proposal from Paul H called them target roots and they got renamed to dependent roots as part of refining how they fit into the REST API, but that was after I did the original draft PR - apparently I missed renaming this when bringing it over.